source/System.cpp

```diff
@@ -268,6 +268,8 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 			trade[value].SetBase(child.Value(valueIndex + 1));
 		else if(key == "object")
 			LoadObject(child, planets);
+		else if(key == "arrival" && child.Size() >= 2)
+			arrivalDistance = child.Value(1);
```

Comment:
It looks like this should be `arrivalDistance = child.Value(valueIndex);` so that events can properly add to and remove from the arrival distance value.